docs/examples/csi_example.md

```diff
@@ -0,0 +1,252 @@
+<h1 align="center">Stateful Application Backup/Restore - MSSQL</h1>
+<h2 align="center">CSI Volume Snapshotting with AWS EBS</h2>
+
+### Prerequisites
+* OADP operator is [installed](/docs/install_olm.md).
+* Create a credentials secret for AWS:
+
+   `oc create secret generic cloud-credentials --namespace openshift-adp --from-file cloud=<CREDENTIALS_FILE_PATH>`
+
+* Make sure your DataProtectionApplication (DPA) CR is similar to below. Note 
```

Comment:
NIT: Just add save this config as `config/samples/oadp_v1alpha1_dpa.yaml` as you are directly asking the user to create DPA isntance using this file OR just mention that create DPA instance using the above config in `Install Velero` step.